src/transformers/tokenization_utils_fast.py

```diff
@@ -155,6 +155,8 @@ def __init__(self, *args, **kwargs):
         # We call this after having initialized the backend tokenizer because we update it.
         super().__init__(**kwargs)
 
+        self._tokenizer.encode_special_tokens = self.split_special_tokens
```

Comment:
Okay, I think we need to either:
- document about the fact that this will change the splitting mode for the rest of the running script
- or we reset it to a previous versions.
Either way, you are right as this is the only way to effectively support this feature from the tokenizers api!

Reply:
I set it here as some models with their own tokenize function won't hit the call to set this.  

About the documenting: I'll add a comment above the line! Where should I update more documentation?